src/main/java/nl/b3p/tailormap/api/persistence/services/FeatureSource.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2023 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package nl.b3p.tailormap.api.persistence.services;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Basic;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OrderColumn;
+
+public class FeatureSource {
```

Comment:
This class should also have an XML-type field that holds the capabilities (if applicable); using an xml field should allow checking the caps version using a simple xpath to determine if a service should be updated (and the update propagated to interested parts)

Reply:
Good idea

Reply:
Decided against this, parse XML capabilities in middleware if needed